pkgs/servers/home-assistant/custom-components/default.nix

```diff
@@ -1,78 +1,79 @@
+  awtrix = callPackage ./awtrix { };
 
-  better_thermostat = callPackage ./better_thermostat {};
+  better_thermostat = callPackage ./better_thermostat { };
+
+  cync_lights = callPackage ./cync_lights { };
 
   dwd = callPackage ./dwd { };
 
-  elevenlabs_tts = callPackage ./elevenlabs_tts {};
+  elevenlabs_tts = callPackage ./elevenlabs_tts { };
 
-  emporia_vue = callPackage ./emporia_vue {};
+  emporia_vue = callPackage ./emporia_vue { };
 
-  epex_spot = callPackage ./epex_spot {};
+  epex_spot = callPackage ./epex_spot { };
 
-  frigate = callPackage ./frigate {};
+  frigate = callPackage ./frigate { };
 
-  govee-lan = callPackage ./govee-lan {};
+  govee-lan = callPackage ./govee-lan { };
 
-  gpio = callPackage ./gpio {};
+  gpio = callPackage ./gpio { };
 
   homematicip_local = callPackage ./homematicip_local { };
 
   indego = callPackage ./indego { };
 
   local_luftdaten = callPackage ./local_luftdaten { };
 
-  localtuya = callPackage ./localtuya {};
+  localtuya = callPackage ./localtuya { };
 
   mass = callPackage ./mass { };
 
-  midea_ac_lan = callPackage ./midea_ac_lan {};
+  midea_ac_lan = callPackage ./midea_ac_lan { };
 
-  midea-air-appliances-lan = callPackage ./midea-air-appliances-lan {};
+  midea-air-appliances-lan = callPackage ./midea-air-appliances-lan { };
 
-  miele = callPackage ./miele {};
+  miele = callPackage ./miele { };
 
-  moonraker = callPackage ./moonraker {};
+  moonraker = callPackage ./moonraker { };
 
-  ntfy = callPackage ./ntfy {};
+  ntfy = callPackage ./ntfy { };
 
-  omnik_inverter = callPackage ./omnik_inverter {};
+  omnik_inverter = callPackage ./omnik_inverter { };
 
-  prometheus_sensor = callPackage ./prometheus_sensor {};
+  prometheus_sensor = callPackage ./prometheus_sensor { };
 
-  samsungtv-smart = callPackage ./samsungtv-smart {};
+  samsungtv-smart = callPackage ./samsungtv-smart { };
 
-  sensi = callPackage ./sensi {};
+  sensi = callPackage ./sensi { };
 
-  smartir = callPackage ./smartir {};
+  smartir = callPackage ./smartir { };
 
-  smartthinq-sensors = callPackage ./smartthinq-sensors {};
+  smartthinq-sensors = callPackage ./smartthinq-sensors { };
 
-  somweb = callPackage ./somweb {};
+  somweb = callPackage ./somweb { };
 
-  spook = callPackage ./spook {};
+  spook = callPackage ./spook { };
 
-  tuya_local = callPackage ./tuya_local {};
+  tuya_local = callPackage ./tuya_local { };
 
   volkswagen_we_connect_id = callPackage ./volkswagen_we_connect_id { };
 
   volkswagencarnet = callPackage ./volkswagencarnet { };
 
-  waste_collection_schedule = callPackage ./waste_collection_schedule {};
+  waste_collection_schedule = callPackage ./waste_collection_schedule { };
 
-  xiaomi_gateway3 = callPackage ./xiaomi_gateway3 {};
+  xiaomi_gateway3 = callPackage ./xiaomi_gateway3 { };
 
-  xiaomi_miot = callPackage ./xiaomi_miot {};
+  xiaomi_miot = callPackage ./xiaomi_miot { };
 
-  yassi = callPackage ./yassi {};
+  yassi = callPackage ./yassi { };
 }
```

Comment:
Annoying diff here is from `nixfmt` so I left the changes, but I can unformat the file and just commit the relevant change on line 14 if preferred.

Reply:
Yes, please only relevant changes.